dlp/snippets/deid/deid_with_exception_list.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package deid
+
+// [START dlp_deidentify_exception_list]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	"cloud.google.com/go/dlp/apiv2/dlppb"
+)
+
+// deidentifyExceptionList creates an exception list for a regular custom dictionary detector.
+func deidentifyExceptionList(w io.Writer, projectID, input, infoType string, dictWordList []string) error {
```

Comment:
nit: you can reduce the input to just what's needed for testing (i.e. `projectID`). 

See here:
https://googlecloudplatform.github.io/samples-style-guide/#minimal-arguments

Reply:
got that, and changed it.